src/style.css

```diff
@@ -0,0 +1,89 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
```

Comment:
don't add padding-top to body
add margin-top to big input and it's enough

Reply:
i use padding in body to avoid margin collapse

Reply:
could you explain, why it is better to use margin-top?
is it a mistake to use padding in body?

Reply:
there will be no margin collapse bc set `margin-top: 0;` so it's better to remove `padding`